webapp/cypress/integration/auth.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Tests for Login Page
+  Scenario: I should be logged in when I enter correct email and password credentials
+    Given I am on the site homepage
+    When I click on "Login" button on the "Home" Page
+    And I enter "admin@admin.pl" into "Email" field on the "Login" Page
+    And I enter "admin" into "Password" field on the "Login" Page
+    And I click on "Login" button on the "Login" Page
+    Then I should be reddirected to "History" Page of "admin@admin.pl" User
+    And Alert "Login successful" should be displayed on "History" Page of "admin@admin.pl" User
+
+  Scenario: Alert should be displayed when I did not enter email and password credentials
+    Given I am on the site homepage
+    When I click on "Login" button on the "Home" Page
+    And I left empty "Email" field on the "Login" Page
+    And I left empty "Password" field on the "Login" Page
+    And I click on "Login" button on the "Login" Page
+    Then Alert "Email or password is invalid" should be displayed on "Login" Page
+
+  Scenario: Alert should be displayed when I did not enter email credential
+    Given I am on the site homepage
+    When I click on "Login" button on the "Home" Page
+    And I left empty "Email" field on the "Login" Page
+    And I enter "admin" into "Password" field on the "Login" Page
+    And I click on "Login" button on the "Login" Page
+    Then Alert "Email or password is invalid" should be displayed on "Login" Page
+
+  Scenario: Alert should be displayed when I did not enter password credential
+    Given I am on the site homepage
```

Comment:
Każdy test powiera kilka tych samych linijek. O ile dobrze pamiętam to ostatnio dodali nowy feature, w którym można dodać sekcje Background, i w niej wstawić te kroki, które mają być wywołane przed każdym testem. https://relishapp.com/cucumber/cucumber/docs/gherkin/background

Reply:
poprawiłem